docs/how_to_guides/quart_extensions.rst

```diff
@@ -25,6 +25,8 @@ here,
   port of Flask-Login to work natively with Quart.
 - `Quart-minify <https://github.com/AceFire6/quart_minify/>`_ minify
   quart response for HTML, JS, CSS and less.
+- _`Quart-Mongo <https://github.com/Quart-Addons/quart-mongo>`_  Bridges Quart, Motor, and Odmantic to create a powerful MongoDB   
```

Comment:
I don't think the preceding `_` is required. Also can you limit the line length as with the others.